src/main/java/org/example/Methods.java

```diff
@@ -0,0 +1,179 @@
+package org.example;
+
+
+import java.util.*;
+import java.util.function.Predicate;
+
+public class Methods {
+
+    public static void State(int number, String player, String[][] board) {
+        switch (number) {
```

Comment:
czyli ktos wybiera numer i to odpowiada punktowi w ktory strzela tak ? 
no to tutaj 2 sugestie :D 
1. Zrobice klase Point ktora trzyma x i y :D 
2. jak zrobicie Mape albo Enum ktory trzyma to ze 1 -> [0][0] to bedziecie mogli latwo z tego korzystac i latwo rozszerzac gre i zniknie wam ten caly switch tylko wystarczy mag.get(1) ktore zwroci Point 0,0